### ISSUE TITLE
Live TV voice filter never returns if filtering fails

### ISSUE THREAD

**Issue by jimdogx** [jellyfin-archive/jellyfin-roku-legacy#878]
**Describe the bug**
If you attempt to filter by a channel you don't have the load spinner never ends. Hitting back does get you out of the situation.

Build 1.6.2 (currently unstable)

**Expected behavior**
I'd expect a No Data / empty TV guide grid if no channel is found.

**Comment by candry7731** [comment]
Is this still happening?

**Comment by jimdogx** [comment]
Yep.  For example, I don't have CNN on my Live TV library.  If I click the mic and say "CNN", it says "Filtered by CNN" and the load spinner never ends.
